tests/unit/core/mender-utils_test.cpp

```diff
@@ -0,0 +1,75 @@
+/**
+ * @file      mender-utils_test.c
+ * @brief     Unit Tests for mender-utils.c
+ *
+ * Copyright Northern.tech AS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <gtest/gtest.h>
+
+extern "C" {
+#include "mender-utils.h"
+#include "mender-client.h"
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+}
```

Comment:
This seems wrong. Our headers should not require this and the standard ones should be used in the C++ way, right?

Reply:
Thanks, I will look into it :thinking:

Reply:
They were not needed - I removed them.